scheduler/metrics.go

```diff
@@ -16,7 +16,7 @@ type TableClientMetrics struct {
 	Resources uint64
 	Errors    uint64
 	Panics    uint64
-	Duration  time.Duration
+	Duration  atomic.Pointer[time.Duration]
```

Comment:
change equal to use load

Reply:
https://github.com/cloudquery/plugin-sdk/pull/1765/files#diff-cb3db60fb46d339e0932dde09ff6854c037d4513602558ae13cd8c1790619e5eR26